### ISSUE TITLE
Make it possible to configure the `intro` property of a component class

### ISSUE THREAD

**Issue by Windvis** [sveltejs/svelte-repl#43]
I noticed that it isn't possible to configure the initialization options of the generated SvelteComponent class. I think most of them aren't really needed to be configurable, but the intro one seems handy.

Do you think it would be a good idea to add something like this?

**Comment by donnisnoni95** [comment]
good idea. but maybe you better make this issue on this repo > [sveltejs/svelte](https://github.com/sveltejs/svelte)

**Comment by Conduitry** [comment]
This is the right repo for this issue. This sounds like on okay request but I'm not sure where the UI should live.

**Comment by pngwn** [comment]
I wonder if there are any other options like this that need to be configurable in the REPL. We could have some more toggles underneath or above the compiler options, I guess?

**Comment by Windvis** [comment]
Yea, I was wondering about this myself. My first idea was to add it below the compiler options as well. Something similar to how the compiler options are shown:

```javascript
import App from './App.svelte';

const app = new App({
  target: document.body, // Hardcoded to whatever the REPL uses now?
  intro: false, // Toggle here
  props: {
    // Not sure if this needs to be configurable
  }
});
```
This is probably the least amount of work. All the UI stuff exists already.

Another option which would make more sense if there will be more REPL options in the future is to add a new "settings" icon to the top bar which would open a settings screen (or dropdown). The compiler options could also be moved there. This would also open up the possibility to add CodeMirror options (VIM mode :angel:).
